dask/tests/test_threaded.py

```diff
@@ -105,58 +103,28 @@ def test_f():
     assert L == [1] * 20
 
 
-code = """
-import sys
-import time
-from dask.threaded import get
-
-def signal_started():
-    sys.stdout.write('started\\n')
-    sys.stdout.flush()
-
-def long_task(x):
-    time.sleep(5)
-
-dsk = {('x', i): (long_task, 'started') for i in range(100)}
-dsk['started'] = (signal_started,)
-dsk['x'] = (sum, list(dsk.keys()))
-get(dsk, 'x')
-"""
-
-
 # TODO: this test passes locally on windows, but fails on appveyor
 # because the ctrl-c event also tears down their infrastructure.
 # There's probably a better way to test this, but for now we'll mark
 # it slow (slow tests are skipped on appveyor).
 @pytest.mark.slow
 def test_interrupt():
+
+    def long_task():
+        sleep(5)
+
+    def interrupt():
+        sleep(0.5)
+        pid = os.getpid()
+        os.kill(pid, signal.SIGINT)
```

Comment:
> Actually, I lied, testing in a task would kindof test the failure case we're looking for, and would be a robust test. Will fix.

Nvm, I was wrong here too. The fix was for a bug where hitting `ctrl-c` while all threads where doing long running tasks would block until one of them finished. This can only be tested while raising a keyboard interrupt in the main thread while waiting on `queue_get`.